### DISCUSSION TITLE
How to deal with the log name with a suffix, such as access.log.20221221314, using access.log.* will load all the logs, I only want to monitor the logs of the current hour

### DISCUSSION THREAD

**Discussion by huozhirui** [google/mtail#885] [Q&A] [answered]
How to deal with the log name with a suffix, such as access.log.20221221314, using access.log.* will load all the logs, I only want to monitor the logs of the current hour

**Comment by jaqx0r** [comment]
Can you use a symlink to the latest log file, that gets updated after the logs are rotated?

**Comment by huozhirui** [comment]
In this way, I need to establish a new soft connection every hour, and for multiple log files, I need to establish multiple maintenance logics, which is a lot of work.

**Comment by jaqx0r** [comment, marked as answer]
Well the normal practice is to use log rotation to create the date-named log files so that there's a common base name to follow.

After a timeout of one hour, all the logs that haven't been written to get removed from the list of watched logs:  https://google.github.io/mtail/Deploying.html#setting-garbage-collection-intervals
